CHANGES.rst

```diff
@@ -1,10 +1,13 @@
 Changelog
 =========
 
-Version 0.3.0 [unreleased]
+Version 0.3.0 [2020-06-26]
 --------------------------
 
-WIP
+- [models] Added swappable models and extensible classes
+- [dependencies] Upgraded ``phonenumbers 8.12.0``,
+  ``openwisp-utils 0.5[rest]`` and ``allauth 0.42.0``
+- [api] Added REST API endpoint for bearer authentication
```

Comment:
I see [some more changes since the last release](https://github.com/openwisp/openwisp-users/compare/0.2.2...master), maybe the points below will help:

- documentation improvements
- Added options `OPENWISP_USERS_AUTH_API` & `OPENWISP_USERS_AUTH_THROTTLE_RATE` to disable and throttle API.
- Added dependencies `redis` & `django-redis` 
- supported version of `djangorestframework` may have changed since the last release, please check that.
- Isn't it `django-allauth 0.42`?
- Add ability to extend the openwisp-users django app

Reply:
Thanks @atb00ker .
- `redis` and `django-redis` are development requirements, I don't think they should go in changelog
- DRF is added as a dependency of `openwisp-utils[rest]`. So, I don't think we need to worry about that here
- `[models] Added swappable models and extensible classes` and `Add ability to extend the openwisp-users django app` are same. What do you think?

I have added documentation improvements and fixed django-allauth.